picocolors.js

```diff
@@ -1,12 +1,13 @@
-let argv = process.argv || [],
-	env = process.env
+let p = process || {},
```

Comment:
```suggestion
let p = globalThis.process || {},
```

I assume that the `|| {}` is so that this package also works when bundled and/or used in a browser.

When doing so, it's not uncommon to accidentally make the code run in strict mode, and in strict mode accessing `process` when it's not defined is an error :)

Reply:
Oh actually using `globalThis` would make this library not work in older Node.js version :/